include/wx/osx/private/uilocale.h

```diff
@@ -0,0 +1,23 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/osx/private/uilocale.h
+// Purpose:     Helper for making pointer to current NSLocale available
+//              for locale-dependent controls under macOS.
+// Author:      Vadim Zeitlin
```

Comment:
Not really :-)

Reply:
I just copied the comments from another file, adjusting the _purpose_ but not paying attention to the other items.

Reply:
Will you change it or should I, when merging?

Reply:
I changed the author name.